### PR TITLE
Add attestation changes to v.0.13 changelog

### PR DESCRIPTION
**Pull request by yentsanglee**
Added attestation changes.

Signed-off-by: Yen Lee <yenlee@microsoft.com>

### CODE REVIEW

**Review by qiucwang — APPROVED** [review]
Thanks for adding that.

**Review by ryanhsu19 — APPROVED** [review]
Clear and concise. Thanks for the work!

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+### Added
+- Enabled oe_verify_evidence() with NULL format id to verify legacy report generated by oe_get_report().
```
Comment (mingweishih) [inline CHANGELOG.md:14]:
```suggestion
- Enabled oe_verify_evidence() with a NULL format id to verify the legacy report generated by oe_get_report().
```

**Review by jianlia — APPROVED** [review]

**Comment by yentsanglee** [comment]
The PR targeted at v.0.13 branch which is wrong. Abandon the PR.

**Pull request closed without merging**